integration/tests/basic/rpc/test_rpc_estimate_gas.py

```diff
@@ -0,0 +1,173 @@
+import typing as tp
+
+import allure
+import pytest
+
+from integration.tests.basic.helpers import rpc_checks
+from integration.tests.basic.helpers.basic import BaseMixin
+from utils import helpers
+
+
+@pytest.fixture(scope="class")
+def common_contract(web3_client, class_account) -> tp.Any:
+    common_contract, _ = web3_client.deploy_and_get_contract(
+        "common/Common", "0.8.12",
+        contract_name="Common", account=class_account
+    )
+    yield common_contract
+
+
+@allure.feature("JSON-RPC-ESTIMATE-GAS validation")
+@allure.story("Verify eth_estimateGas RPC call")
+class TestRpcEstimateGas(BaseMixin):
+    account: "eth_account.signers.local.LocalAccount" = None
+
+    @pytest.fixture(params=[(850000, 15000), (8500000, 150000), (8500000, 150000)])
+    def constructor_args(self, request: tp.Any) -> tp.List[int]:
```

Comment:
why 3 times the same values and why you need this fixture?

Reply:
The same as here https://github.com/neonlabsorg/neon-tests/pull/159#discussion_r1356482754

Reply:
Rewritten